interfaces/api/v1/reader/__init__.py

```
@@ -0,0 +1,208 @@
+                novel_id=novel_id,
+                chapter_number=chapter_number,
+                overall_readability=report.overall_readability,
+                chapter_hook_strength=report.chapter_hook_strength,
+                pacing_verdict=report.pacing_verdict,
+                avg_scores=report._compute_avg_scores(),
+                feedbacks_json=json.dumps(
+                    [f.to_dict() for f in report.feedbacks],
+                    ensure_ascii=False,
+                ),
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)